examples/workloads/neuron.yaml

```diff
@@ -21,9 +21,9 @@ spec:
         name: neuron
         resources:
           limits:
-            aws.amazon.com/neuron: "1"
+            aws.amazon.com/neurondevice: "1"
```

Comment:
`neurondevice` type is being deprecated as part of latest neuron SDK release and I would not add support for it.

https://awsdocs-neuron.readthedocs-hosted.com/en/latest/general/announcements/neuron2.x/announce-no-support-device-version.html

Reply:
thats an odd and unfortunate thing to hear - device is what is used throughout the documentation:
- https://awsdocs-neuron.readthedocs-hosted.com/en/latest/general/arch/neuron-hardware/inferentia2.html
- https://awsdocs-neuron.readthedocs-hosted.com/en/latest/general/arch/neuron-hardware/trainium.html

do we have documentation that equates a "neuron" to "neuron cores"?

Reply:
In workload yamls, going forward
- `aws.amazon.com/neuron` is used for requesting neuron devices
- `aws.amazon.com/neuroncore` is used for requesting neuron cores

Previously both `neuron` & `neurondevice` were used for requesting devices.

Reply:
very confusing, but ok

Reply:
removed in https://github.com/aws/karpenter-provider-aws/pull/6510/commits/d2ee76c03077330391b39a935ff08fdc7720c0d7